packages/content/src/hub.ts

```diff
@@ -9,6 +9,7 @@ import {
 import { itemToContent } from "./portal";
 import { isSlug, addContextToSlug, parseDatasetId } from "./slugs";
 import { enrichContent, IFetchEnrichmentOptions } from "./enrichments";
+import { isExtentCoordinateArray } from "../../common/src/extent";
```

Comment:
When we import across package boundaries we should import from the package name:

```suggestion
import { isExtentCoordinateArray } from "@esri/hub-common";
```